main/lsp/lsp.cc

```diff
@@ -29,7 +29,7 @@ LSPLoop::LSPLoop(unique_ptr<core::GlobalState> gs, const options::Options &opts,
     rootPath = opts.rawInputDirNames.at(0);
 }
 
-variant<LSPLoop::TypecheckRun, pair<unique_ptr<ResponseError>, unique_ptr<core::GlobalState>>>
+variant<LSPLoop::QueryRun, pair<unique_ptr<ResponseError>, unique_ptr<core::GlobalState>>>
```

Comment:
can we get rid of the `pair`? I always forget what it stands for

Reply:
Just pushed a commit that moves the response error into `QueryRun`.